src/java/src/com/facebook/csslayout/CSSNode.java

```diff
@@ -400,4 +400,20 @@ public void setDefaultPadding(int spacingType, float padding) {
       dirty();
     }
   }
+
+  /**
+   * Resets this instance to its default state. This method is meant to be used when
+   * recycling {@link CSSNode} instances.
+   */
+  public void reset() {
+    style.reset();
+    layout.reset();
+    lineIndex = 0;
+    mLayoutState = LayoutState.DIRTY;
+    mMeasureFunction = null;
```

Comment:
The measure function is not something that is set by the client / layout, at least not in RN. Not sure it makes sense to reset it.